app/src/main/java/eu/kanade/tachiyomi/data/export/LibraryExporter.kt

```diff
@@ -0,0 +1,66 @@
+package eu.kanade.tachiyomi.data.export
+
+import android.content.Context
+import android.net.Uri
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+import tachiyomi.domain.manga.model.Manga
+
+object LibraryExporter {
+
+    data class ExportOptions(
+        val includeTitle: Boolean,
+        val includeAuthor: Boolean,
+        val includeArtist: Boolean,
+    )
+
+    fun exportToCsv(
```

Comment:
Do `scope.launch` from UI side
```suggestion
    suspend fun exportToCsv(
```